varnish-sys/Cargo.toml

```diff
@@ -16,6 +16,7 @@ readme.workspace = true
 name = "varnish_sys"
 
 [package.metadata.libvarnishapi]
+# FIXME: which version should be set here, and what uses it?
```

Comment:
it can be used by downstream projects: https://github.com/varnish-rs/vmod-rers?tab=readme-ov-file#all-platforms

options:
- remove it, and scrub the downstream READMEs
- leave it here and replace the FIXME with a more relevant comment

Reply:
this is not the way to do it. There is a proper link-based system.

Define the name using `links`:

https://github.com/varnish-rs/varnish-rs/blob/f870c7bebaebc9f5c5aa5ee4308dbb8f1df880ca/varnish-sys/Cargo.toml#L4

Set any values using `cargo::metadata` in the build script:

https://github.com/varnish-rs/varnish-rs/blob/f870c7bebaebc9f5c5aa5ee4308dbb8f1df880ca/varnish-sys/build.rs#L83

Using the env var in any build script downstream:

https://github.com/varnish-rs/varnish-rs/blob/f870c7bebaebc9f5c5aa5ee4308dbb8f1df880ca/varnish/build.rs#L5

Reply:
sure, but we can do it afterwards, as this is all versioned - and that's what versions are for :)

Reply:
yes, but I'd rather not add a `FIXME` asking for information that you now have :-)

Reply:
of course - i will remove it :)

BTW, the only realistic way to get that value AFTER the binary is built is to create some temporary file in the output:

```rust
// build.rs
fn main() {
    let version = std::env::var("DEP_VARNISHAPI_VERSION_NUMBER").unwrap();
    
    let out_dir = std::env::var("OUT_DIR").unwrap();
    let dest_path = std::path::Path::new(&out_dir).join("../../../build_metadata.json");
    std::fs::write(
        dest_path,
        format!(r#"{{"varnish_version":"{version}"}}"#)
    ).unwrap();
}
```